Makefile

```diff
@@ -60,3 +60,43 @@ load-test-server-local:
 .PHONY: echo-var
 echo-var:
 	@echo "$($(VAR))"
+
+
+## Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
+## Tool Binaries
+CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
+CONTROLLER_TOOLS_VERSION ?= v0.17.2
```

Comment:
Why not 0.17.3? Wouldn't 0.19.0 be the better choice, using the same minor release of Kubernetes as the rest of this repo?

Reply:
I noticed that WVA project also used v0.17.2, but changed to v0.17.3 should be fine.  I will do this.Thanks.